frontend/revive-gallery-app/src/App.js

```diff
@@ -14,6 +15,7 @@ function App() {
           <Route path="/login" element={<LoginForm />} />
           <Route path="/register" element={<RegistrationForm />} />
           <Route path="/mp" element={<Marketplace />} />
```

Comment:
@SaiKiran0407 It's better to change the path to "/products", because it makes more sense than "/mp".

Reply:
@anjaliputta Sure, I will update it accordingly. Thank you